frontend/src/components/resources/resource_show.jsx

```diff
@@ -32,7 +34,17 @@ class ResourceShow extends React.Component {
             </button>
             <p>{resource.featured}</p>
             <p>{resource.description}</p>
-            <p>belongs to habit {habit.title}</p>
+
+            <Link 
+              style={{
+                'color': 'red',
+                'display': 'block',
+                'margin-top': '5px',
+              }}
+              to={`/habits/${resource.habit}`}>
+              return to habit {habit.title}
```

Comment:
since we can access the resource show page from the resource index, would it make sense to have it say
"belongs to" still?

Looks good though!

do you prefer inline styling?

Reply:
changed to "belongs to". Inline styling is cleaner to me if not intended to be reused